tests/testAutopas/tests/selectors/AutoTunerTest.cpp

```diff
@@ -82,6 +82,10 @@ TEST_F(AutoTunerTest, testAllConfigurations) {
   //                        vlp_c01                     (AoS <=> SoA, noNewton3)                             = 2
   configsPerContainer[autopas::ContainerOption::pairwiseVerletLists] = 18;
 
+  // Octree:                ot_c01                      (AoS <=> SoA, noNewton3)                             = 2
+  //                        ot_naive                    (disabled)                                           = 0
```

Comment:
why disabled?

Reply:
As discussed, I added a new issue and referenced it in a comment at that place in the code in dc8f1dc.